a3p-integration/proposals/n:upgrade-next/submitBid.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+import {
+  GOV1ADDR,
+  CHAINID,
+  agd,
+  agopsInter,
+  addUser,
+  waitForBlock,
+  provisionSmartWallet,
+  ATOM_DENOM,
+} from '@agoric/synthetic-chain';
+
+export const bankSend = (from, addr, wanted) => {
+  const chain = ['--chain-id', CHAINID];
+  const fromArg = ['--from', from];
+  const testKeyring = ['--keyring-backend', 'test'];
+  const noise = [...fromArg, ...chain, ...testKeyring, '--yes'];
+
+  return agd.tx('bank', 'send', from, addr, wanted, ...noise);
+};
+
+const bidder = await addUser('long-living-bidder');
+console.log('BIDDDER', bidder);
```

Comment:
spelling

Reply:
Resolved in https://github.com/Agoric/agoric-sdk/pull/10229/commits/990126165e1c8ba2ac7cc8a52ad696cf7fc347a3